test/dlc_tests/sanity/test_generate_coverage_doc.py

```diff
@@ -9,8 +9,8 @@
 from test.test_utils import LOGGER, is_mainline_context
 from test.test_utils.test_reporting import get_test_coverage_file_path
 
-
-TEST_COVERAGE_REPORT_BUCKET = f"dlc-test-coverage-reports"
+account_id = boto3.client('sts').get_caller_identity().get('Account')
```

Comment:
Why not use $ACCOUNT_ID which we have included in our documentation instead of the sts call?

Reply:
Why use $ACCOUNT_ID when we know that the sts call will always work, and will always be reliable?

Reply:
Main reason is just grabbing from an env variable is faster, and we can always default to sts

Reply:
I don't think it would make much of a difference, but sure, I wouldn't mind seeing it implemented as something like
```
account_id = os.getenv("ACCOUNT_ID", boto3.client("sts").get_caller_identity().get("Account"))
```